src/components/Token.vue

```diff
@@ -6,7 +6,7 @@
 
 <script>
 import { formatBytes32String } from '@ethersproject/strings';
-
+import { getUrl } from '@snapshot-labs/snapshot.js/src/utils.ts';
```

Comment:
```suggestion
import { getUrl } from '@snapshot-labs/snapshot.js/src/utils.ts';

```

Reply:
What does this do?

Reply:
Just add a line between import and export